cves/CVE-2012-0022.yml

```diff
@@ -132,8 +137,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: connectors
```

Comment:
In the answer field, please indicate why you thought connectors was an appropriate name for the subsystem.